pkgs/top-level/all-packages.nix

```diff
@@ -21994,8 +21994,6 @@ in
     pythonPackages = python3Packages;
   };
 
-  way-cooler = callPackage ../applications/window-managers/way-cooler {};
```

Comment:
I'd like to have an error message explaining the removal here with `way-cooler = throw "..."`